src/style.css

```
@@ -1 +1,38 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  width: max-content;
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+
+}
+
+.stars--1 :first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url(images/star-active.svg)
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url(images/star-active.svg)
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg)
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg)
+}
```

```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url(images/star-active.svg)
}
```

Combine selectors using comma